content/cohorts/vr-ar/scioxr.en.adoc

```diff
@@ -0,0 +1,36 @@
+---
+title: ScioXR
+weight: 50
+categories: "team"
+tags: ["team"]
+type: profiles
+
+description: Platform for teachers to make tailored learning STEM content content with VR.
```

Comment:
There is a double "content" word

Reply:
Fixed in commit eb0117ddf14c9db2022f651eaec734d08a3a9df6.